onboard-julia/App.tsx

```diff
@@ -6,11 +6,30 @@
  */
 
 import React from 'react';
-
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
 import {Login} from './login-page';
+import {View, Text} from 'react-native';
+
+const Stack = createNativeStackNavigator();
+
+function NewScreen() {
+  return (
+    <View>
+      <Text>New Screen</Text>
+    </View>
+  );
+}
 
-function App(): React.JSX.Element {
-  return <Login />;
+function App() {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name="Login" component={Login} />
+        <Stack.Screen name="Blank" component={NewScreen} />
+      </Stack.Navigator>
+    </NavigationContainer>
+  );
 }
```

Comment:
Aqui vc está criando a stack de navegação do app no componente App, seria legal manter separado essa stack da criação dos componentes de páginas. É bom que quando for alterar a tela já vai estar separada e isolada